routes/curations.js

```diff
@@ -54,8 +54,41 @@ router.get(
   asyncMiddleware(async (request, response) => {
       const coordinates = utils.toEntityCoordinatesFromRequest(request)
       curationService.list(coordinates).then(function(result){
+
+        // access the request url and method
+        const { method, url } = request;
+        console.log('method', method);
+        console.log('url', url);
+
+        //trim and split the request url for processing
+        let trimmed = url.trim();
+        let myStringArray = trimmed.split("/");
+        let text_file_name = "./test/fixtures/dummy_data_curations_"
+
+        //iterate through the 'url array' and create a filename string from it
+        const arrayLength = myStringArray.length;
+        for (var i = 0; i < arrayLength; i++) {
```

Comment:
@moranthomas we should really consider using let in place of var, especially inside for loops. No need to optimize this now tho.